csharp/GildedRoseTest.cs

```diff
@@ -12,7 +12,130 @@ public void foo()
             IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
             GildedRose app = new GildedRose(Items);
             app.UpdateQuality();
-            Assert.AreEqual("fixme", Items[0].Name);
+            Assert.AreEqual("foo", Items[0].Name);
+        }
+
+        [Test]
+        public void AfterSellByDate_QualityDegradesTwiceAsFast()
+        {
+            IList<Item> items = new List<Item> { new Item { Name = "Big Mac", SellIn = 5, Quality = 50 } };
+            GildedRose app = new GildedRose(items);
```

Comment:
You can use a `SetUp` and/or `TearDown` method for initializing/tearing down things.

Reply:
You can add methods to the GildedRose class to add/remove items